cloud/scope/common.go

```diff
@@ -23,6 +27,7 @@ import (
 const (
 	// defaultClientTimeout is the default timeout for a client Linode API call
 	defaultClientTimeout = time.Second * 10
+	maxBody              = 131072
```

Comment:
should have a comment here like above for what this does

Reply:
Done